### PR TITLE
Fix the alignment of the up speak page's tablet view

### LINKED ISSUE #1336: Fix the alignment of the upspeak page's tablet view
**Describe the bug**
The alignment of the Upspeak page's tablet view is ugly because there is lot of space above and below the image. Also, the text is aligning to the left side of the page so much.

**To Reproduce**
1. Go to 'https://sefglobal.org/upspeak/'
2. Click on 'inspect'
3. Scroll down 

**Expected behavior**
It should look like what we see in the mobile view.

**Screenshots**
image
image

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome

**Additional context**
N/A

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by VishwaAmarasinghe**
## Purpose

The purpose of this PR is to fix #1336 

## Goals

To correct the alignment of this page's tablet view .

## Approach

Changed some bootstrap commands.

### Screenshots

image

  
### Preview Link

https://pr-1337-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: upspeak/index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
 <section class="section pb-10 bg-gradient-white">
     <div class="container-fluid">
         <div class="row row-grid align-items-center justify-content-center">
-            <div class="col-md-4">
+            <div class="col-lg-5">
```
Comment (anjula-sack) [inline upspeak/index.html]: We need to keep `col-md-4` as well.
```suggestion
            <div class="col-lg-5 col-md-4">
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — CHANGES_REQUESTED** [review]
`col-lg-5` make it bigger for the desktop view. Try comparing these two,

https://pr-1337-sef-site.surge.sh/upspeak/
https://sefglobal.org/upspeak/

**Comment by VishwaAmarasinghe** [comment]
If we still use col-md-4 it will be a problem in tablet view.
image

**Review by anjula-sack — APPROVED** [review]
Good job! @VishwaAmarasinghe

**Pull request merged**